app/Main.hs

```diff
@@ -13,6 +13,11 @@ module Main ( main ) where
 
 -- | 'main' is the main entry point for this application.
 main :: IO ()
-main = putStrLn "Hello World!"
+main = do
+    -- This prevents delays in outputting prompts and other strange issues
+    hSetBuffering stdout NoBuffering
```

Comment:
I think you need to import `System.IO` for this

Reply:
Yep, sorry, I was too busy worrying about creating a draft pull request I forgot that bit. Fixed with 
https://github.com/fpclass/perpetual-haskelling-initiative/pull/7/commits/497c1b6521bdc53e60149a909cfc48e1a5686227